### ISSUE TITLE
Suggestions for 20 New Permament Dark events 

### ISSUE THREAD

**Issue by Grobobobo** [long-war-2/lwotc#898]
Some tactical upgrades need their pool extended for wotc enemies, like advent ones should have priest and purifier, spectres on some mechanical

Some WOTC dark events like barrier need to be updated for LW2 enemies

Anyway, you asked, I said 20, I’m giving 20

**Early Game (around 5-10 FL?) low-ish impact**
1.	Tactical Upgrades: Fortress
Priests now gains fortress.
100% chance
2.	Tactical Upgrades: Regeneration
Spectres now gain regeneration, 3hp
100% chance
3.	Tactical Upgrades: Return fire
Purifiers (after I add the pistol to them) now gain return fire
100% chance
4.	Tactical Upgrades: Precise
Advent primary weapons now ignore up to 10 target dodge
Either 33% chance on all ADVENT or 100% on troopers
5.	Tactical Upgrades: Salvo
Advent Grenadiers and some mutons gain the salvo ability
100% grenadiers, 33% mutons 
(there is some possibility for some AI tweaks since the engineer can possible flashbang the same thing twice? Will need to look into it a bit)
6.	Tactical Upgrades: Flanker
Advent troopers gain the flanker ability, giving them 10% aim and 10% crit against flanked units
100% on advent troopers
7.	Tactical Upgrades: Headshot
Advent can gain the headshot ability, which refunds their actions on a lost kill
50% all advent ??? actually not sure about that one
8.	Tactical Upgrades: Hardening
Sectoids lose the “meele vulnerability” passive
	100% sectoids
9.	Tactical Upgrades: Phosphorus
Purifiers gain phosphorus
100% purifier
10.	Tactical Upgrades: Bind awareness
Vipers gain +15 defence when strangling
100% vipers

**Late game, (>10 FL) high-ish  impact**
1.	Tactical Upgrades: Muton Bladestorm
Mutons now gain bladestorm
100% mutons
2.	Tactical Upgrades: Stun lancer bladestorm
Stun lancers now have bladestorm
100% stun lancers
3.	Tactical Upgrades: Bastion
Priests now gain bastion
Possibly add the fortress one as a requirement
4.	Tactical Upgrades: Intimidate
Some mutons and berserkers can now gain the intimidate ability
25% mutons, 50% berserkers
5.	Tactical Upgrades: Precision claws
Chryssalid meele attacks can now ignore up to 20 of the target dodge
100% all lids
6.	Tactical Upgrades: lid Close and Personal 
Chryssalids now gain close and personal
100% all lids (I’m 99% sure extended info lies and Close and personal bonus uses the unit’s original position in meele) 
7.	Tactical Upgrades: Bull Rush
Berserkers gain the bull rush ability
100% zerkers
8.	Tactical Upgrades: Napalm-X
Purifiers get napalm-X
100% purifiers
9.	Tactical Upgrades: Holotargeting
Some advent get Holotargeting
33% all advent
10.	Tactical Upgrades: Kineting plating
Shieldbearers now gain kinetic plating
100% all shieldbearers

**Comment by Anxoczt** [comment]
Interesting, although i wouldnt give regular Mutons intimidation and just keep that to berserkers so players are encouraged to hit hard rather than paper cutting them to death. Intimidation on regular mutons seems abit excessive.

And i have few ideas too
1) Agent Epsilon (tactical upgrade)
All viper species poison can cause unconsciousness based on target's will with every damage proc.
(With the change that you can resurrect unconscious troops without resources, this would increase threat from getting poisoned but wouldn't cause team ravaging consequences if you do win the fight)
2) Nanite Saboteurs (tactical upgrade)
Spectres plasma weapons jam targets weapon if they hit. Forcing the XCOM trooper to reload.
3) Ravager rounds (tactical upgrade)
All Chosen receives sort of "Bluescreen" round since ever since their nerf to their weapons, they became hard-countered by Sparks who can just eat continuous attacks due to large HP pool, armour and immunity to dazing effects.
4) Hex Rounds (Tactical upgrade)
Sectoids and Priests have 30% bonus critical hit chance towards psionic XCOM troops.
5) Experiments (Strategic DE)
Increases sitrep chance by 20%
6) Plasma Fire (Tactical upgrade ?)
All plasma weapons have a 20% chance to ignite a target that is hit by them.

**Comment by SirMrMister** [comment]
You can have 14 already implemented ones from [More Dark Events: Enemy abilities](https://steamcommunity.com/workshop/browse/?appid=268500)
Extract:
ADVENT:
ADVENT Targeting Uplink: After an ADVENT soldier hits a target with an attack, all nearby ADVENT soldiers gain bonuses to aim and critical chance.
ADVENT Defensive Tactics: When an ADVENT soldier dies, all other nearby ADVENT soldiers gain bonuses to defense and dodge.

Robots:
Electromagnetic Shielding: While armored, all ADVENT robotic units are immune to stun and freeze effects and gain increased hack defense.
Adaptive Servos: All ADVENT robotic units receive improved adaptive actuators. After losing all armor, they gain bonuses to mobility, aim and defense.

Vipers:
Malicious Instincts: All Vipers receive bonuses to aim and damage on attacks against targets suffering from Burning, Poison, Acid, or Bleeding.
The damage bonus only applies to direct attacks, not damage-over-time effects.
Mutated Glands: All Vipers deal increased damage with abilities that deal poison, acid, fire or psionic damage.

Sectoids:
Solace: All Sectoids gain the Solace ability, shielding themselves and nearby allies from all harmful mental effects.
Battle Premonition: All Sectoids gain the Battle Premonition ability. Units with the Battle Premonition ability do not trigger reaction fire. While healthy and unimpaired, they receive massive bonuses to dodge and defense.
Lowering the Sectoid's health below a certain threshold or disorienting / stunning them will remove the dodge / defense bonus.

Berserkers:
Intimidating Presence: All Berserkers gain the Intimidating Presence ability, lowering the aim, mobility and will of all organic enemies in their vicinity.
Does not apply to robotic units or soldiers who are immune to mental status effects.
Unstoppable Force: All Berserkers gain the Unstoppable Force ability, making them immune to poison, stun, Chryssalid poison and mental status effects.

Mutons:
Combat Hardened: All Mutons gain the Combat Hardened ability, making them highly resistant to explosive damage and critical hits.
Close Combat Specialist: All Mutons gain the Close Combat Specialist ability, granting them a free reaction shot with their primary weapon at any visible enemy within four tiles who moves or fires.

Chryssalids :
Eviscerate: All Chryssalids receive large bonuses to aim and crit chance when attacking wounded targets. Melee attacks against organic targets heal the Chryssalid for a percentage of damage dealt.
Nimble Claws: After hitting an enemy with a melee attack, Chryssalids gain bonus mobility. After taking damage, they gain bonus dodge.

**Comment by pledbrook** [comment]
We should consider adding quite a few permanent tactical dark events and add a second wave option that makes them more common.

**Comment by pledbrook** [comment]
I don't think this is needed for 1.0, but these are useful suggestions to keep around.
